surfkit/cli/main.py

```diff
@@ -141,6 +141,9 @@ def create_device(
     image: Optional[str] = typer.Option(
         None, help="The image to use for the desktop. Defaults to Ubuntu Jammy."
     ),
+    region: Optional[str] = typer.Option(
+        "us-east-1", help="AWS region. Defaults to 'us-east-1'."
```

Comment:
Sorry one last thing, can we add the options here?

Reply:
ah - sure!